.changeset/popular-planes-tap.md

```diff
@@ -0,0 +1,27 @@
+---
+"@comet/blocks-admin": major
+"@comet/cms-admin": major
+"@comet/blocks-api": major
+"@comet/cms-site": major
+"@comet/cms-api": major
+---
+
+Revise video blocks.
+
+- add preview image to video blocks
+- move `YouTubeVideoBlock` to `@cms` packages
```

Comment:
Technically, this are two changes (the first being minor, the second being major). Should we split it into two changesets?